services/leaderboard/src/main/java/io/openliberty/spacerover/leaderboard/rest/Leaderboard.java

```diff
@@ -148,44 +160,21 @@ public Response clear() {
 
 	}
 
-	@Consumes(MediaType.APPLICATION_JSON)
 	@Produces(MediaType.APPLICATION_JSON)
-	@APIResponse(responseCode = "200", description = "Deleted user entries from the leaderboard.")
-	@APIResponse(responseCode = "500", description = "Failed to delete user entry from the leaderboard.")
-	@Operation(summary = "Deletes a user's entries from the leaderboard.")
+	@APIResponse(responseCode = "200", description = "Successfully cleared the leaderboard.")
+	@APIResponse(responseCode = "500", description = "Failed to clear the leaderboard.")
+	@Operation(summary = "Clear the leaderboard entries from the database.")
```

Comment:
fix the descriptions, since we are only clearing a single entry from the leaderboard with this endpoint.